src/Libraries/PythonNodeModelsWpf/IronPythonCompletionProvider.cs

```diff
@@ -97,116 +125,154 @@ public IronPythonCompletionProvider()
             {
                 try
                 {
-                    scope.Engine.CreateScriptSourceFromString("import clr\n", SourceCodeKind.Statements).Execute(scope);
-
                     var libGImports =
-                        "import clr\nclr.AddReference('ProtoGeometry')\nfrom Autodesk.DesignScript.Geometry import *\n";
+                        "clr.AddReference('ProtoGeometry')\nfrom Autodesk.DesignScript.Geometry import *\n";
 
-                    scope.Engine.CreateScriptSourceFromString(libGImports, SourceCodeKind.Statements).Execute(scope);
+                    engine.CreateScriptSourceFromString(libGImports, SourceCodeKind.Statements).Execute(scope);
+                    clrModules.Add("ProtoGeometry");
                 }
                 catch (Exception e)
                 {
                     Log(e.ToString());
-                    Log("Failed to load ProtoGeometry types for autocomplete.  Python autocomplete will not see Autodesk namespace types.");
+                    Log("Failed to load ProtoGeometry types for autocomplete. Python autocomplete will not see Autodesk namespace types.");
                 }
             }
-
+            
+            string pythonLibDir = System.IO.Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.ProgramFilesX86),
+                                                         "IronPython 2.7\\Lib");
+            if (System.IO.Directory.Exists(pythonLibDir) )
+            {
+                try
+                {
+                    var pyLibImports = String.Format("import sys\nsys.path.append(r'{0}')\n", pythonLibDir);
+                    engine.CreateScriptSourceFromString(pyLibImports, SourceCodeKind.Statements).Execute(scope);
+                }
+                catch (Exception e)
+                {
+                    Log(e.ToString());
+                    Log("Failed to register IronPython's native library. Python autocomplete will not see standard modules.");
+                }
+            }
+            
+            else
+            {
+                Log("Valid IronPython installation not found. Python autocomplete will not see native modules.");
+            }
         }
 
         /// <summary>
-        /// Generates completion data for the specified text, while import the given types into the 
+        /// Generates completion data for the specified text, while import the given types into the
         /// scope and discovering variable assignments.
         /// </summary>
-        /// <param name="line">The code to parse</param>
+        /// <param name="code">The code to parse</param>
+        /// <param name="expand">Optionally recurse and extend the search to the entire namespace</param>
         /// <returns>Return a list of IronPythonCompletionData </returns>
-        public ICompletionData[] GetCompletionData(string line)
+        public ICompletionData[] GetCompletionData(string code, bool expand=false)
```

Comment:
we can't change this methods signature at this time - even adding an optional parameter will break binary compatibility:
https://stackoverflow.com/questions/1456785/a-definitive-guide-to-api-breaking-changes-in-net

It's very easy to miss cases here. :(